helm/reana/templates/reana-message-broker.yaml

```diff
@@ -41,8 +43,15 @@ spec:
           name: tcp
         - containerPort: 15672
           name: management
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/rabbitmq/mnesia
       {{- if .Values.node_label_infrastructure }}
       {{- $full_label := split "=" .Values.node_label_infrastructure }}
       nodeSelector:
         {{ $full_label._0 }}: {{ $full_label._1 }}
       {{- end }}
+      volumes:
+      - name: data
+        hostPath:
+          path: /var/lib/rabbitmq/mnesia
```

Comment:
The fix is wrong, the host path should be `/var/reana/...`:

```diff
diff --git a/helm/reana/templates/reana-message-broker.yaml b/helm/reana/templates/reana-message-broker.yaml
index f0786ce..7c49f3e 100644
--- a/helm/reana/templates/reana-message-broker.yaml
+++ b/helm/reana/templates/reana-message-broker.yaml
@@ -52,4 +52,4 @@ spec:
       volumes:
       - name: data
         hostPath:
-          path: /var/lib/rabbitmq/mnesia
+          path: /var/reana/rabbitmq/mnesia

```

Reply:
Fixed, thanks for the catch. 👍